spec/unit/cli_spec.rb

```diff
@@ -22,6 +22,7 @@
 require "chef/telemeter/sender"
 require "chef_apply/ui/terminal"
 require "chef_apply/action/generate_temp_cookbook"
+require "chef_apply/action/install_chef"
```

Comment:
why did we require this? are we using any of the methods from this?

Reply:
Thats just for testing some cases in my local, removed it.